patches/server/0005-Paper-config-files.patch

```diff
@@ -1594,6 +1594,13 @@ index 0000000000000000000000000000000000000000..a33de97340f14219291c4175e9194914
 +            public boolean tick = true;
 +        }
 +
++        public Sniffer sniffer;
++
++        public class Sniffer extends ConfigurationPart {
++            public int hatchTime = 24000;
++            public int boostedHatchTime = 12000;
++        }
```

Comment:
This is more optional, but I've really been becoming a fan of using vanilla constants whenever possible. So maybe use 2 ATs to make the 2 constants for these tick times in SnifferEggBlock public, and reference them here instead. It serves 2 purposes, if the time changes, it auto updates, and if the field is removed, then it conflicts/causes a compile error making someone look at the changes